src/algebraic_geometry/presheafed_space/has_colimits.lean

```diff
@@ -94,16 +94,32 @@ def pushforward_diagram_to_colimit (F : J ⥤ PresheafedSpace C) :
     (pushforward_eq (colimit.w (F ⋙ PresheafedSpace.forget C) f) (F.obj j).presheaf).hom).op,
   map_id' := λ j,
   begin
+    sorry,
     apply (op_equiv _ _).injective,
```

Comment:
Expanding out all these proofs is not the solution to whatever the problem is here. :-(

Reply:
Yeah, it doesn't seem to be going anywhere. I don't have a clue where to start looking for a better fix, though.